code/game/objects/effects/anomalies.dm

```diff
@@ -421,6 +423,83 @@
 			if(EXPLODE_LIGHT)
 				SSexplosions.lowturf += T
 
+/obj/effect/anomaly/delimber
+	name = "delimber anomaly"
+	icon_state = "delimber_anomaly"
+	aSignal = /obj/item/assembly/signaler/anomaly/delimber
+	immortal = TRUE
+	/// Time passed since the last effect, increased by delta_time of the SSobj
+	var/ticks = 0
+	/// How many seconds between each anomaly pulses
+	var/releasedelay = 15
```

Comment:
I think a cooldown would be better but this is fine, as long as you convert it to deciseconds